Scripts/Runtime/Core/ScriptableObjectCollection.cs

```diff
@@ -331,6 +349,14 @@ public void RefreshCollection()
             string folder = Path.GetDirectoryName(assetPath);
             string[] guids = AssetDatabase.FindAssets($"t:{collectionItemType.Name}", new []{folder});
 
+            HashSet<ScriptableObjectCollection> neighbors =
+                AssetDatabase
+                   .FindAssets($"t:ScriptableObjectCollection", new[] { folder })
```

Comment:
Personal preference, but when using the name of a type in a string I find it nice to use `nameof` so it's not hard-coded and will refactor correctly if you rename the class. I don't think it's necessary here but I find it a good habit to just do it everywhere.

```suggestion
                   .FindAssets($"t:{nameof(ScriptableObjectCollection)}", new[] { folder })
```